pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -31,7 +31,14 @@ const (
 
 	// CustomResourceDefinitionSchemaVersion is semver-conformant version of CRD schema
 	// Used to determine if CRD needs to be updated in cluster
-	CustomResourceDefinitionSchemaVersion = "1.22"
+	//
+	// Maintainers: bump minor for each Cilium minor version released and set
+	// patch to .1.
```

Comment:
Why do we need to bump it if the schema is the same as a previous release?

Reply:
@joestringer I noticed that v1.6.10 will reach 1.16 which will conflict with all 1.7.0-1.7.2 releases so it's to prevent such cases.

Reply:
Makes sense.